### PR TITLE
build: typed scripts

### PR DESCRIPTION
**Pull request by jobvs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "scripts": {
         "postinstall": "lerna bootstrap",
         "format": "pretty-quick",
-        "lint": "lerna run lint",
+        "lint": "npm run lint:scripts && lerna run lint",
+        "lint:scripts": "tslint --project scripts/tsconfig.json --config scripts/tslint.json",
```
Comment (diego-antonelli) [inline package.json]: What about use ESLint instead?

https://eslint.org/blog/2019/01/future-typescript-eslint

File: scripts/types/gulp-xml2js.d.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export = index;
```
Comment (diego-antonelli) [inline scripts/types/gulp-xml2js.d.ts]: first declare, then export?

File: scripts/types/rollup-plugin-babel.d.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+declare module "rollup-plugin-babel" {
+    export = index;
```
Comment (diego-antonelli) [inline scripts/types/rollup-plugin-babel.d.ts]: same here

File: package.json (hunk 1 above)
Comment (jobvs) [reply package.json]: In the future, definitely. As of now it's not yet on par with tslint.

File: scripts/types/gulp-xml2js.d.ts (hunk 2 above)
Comment (jobvs) [reply scripts/types/gulp-xml2js.d.ts]: Done

File: scripts/types/rollup-plugin-babel.d.ts (hunk 3 above)
Comment (jobvs) [reply scripts/types/rollup-plugin-babel.d.ts]: Done

**Pull request merged**